### DISCUSSION TITLE
How to limit the number of runtime folders?

### DISCUSSION THREAD

**Discussion by Tonsterr** [cefsharp/CefSharp#4371] [Q&A]
Hi, I have an application (.net 7) using CefSharp.netcore. When the app compiles, there is a runtime folder that get created in the the application's bin\debug directory. And this runtime folder contains 3 subfolders:
![image](https://user-images.githubusercontent.com/3620870/213487297-3a491c42-7153-4e78-bd5f-4ce9d95eb0bb.png)

The application is targeting win 10 x64 systems, so i would only need the win-x64 subfolder. How do I go about setting up CefSharp so that it doesn't create those unneeded folders?

Thanks

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/wiki/Quick-Start-For-MS-.Net-5.0-or-greater#20-set-a-runtimeidentifier-in-your-proj-file
